pkgs/development/ocaml-modules/secp256k1-internal/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitLab
+, buildDunePackage
+, gmp
+, dune-configurator
+, cstruct
+, bigstring
+, alcotest
+, hex
+}:
+
+buildDunePackage rec {
+  pname = "secp256k1-internal";
+  version = "0.2";
+  src = fetchFromGitLab {
+    owner = "nomadic-labs";
+    repo = "ocaml-secp256k1-internal";
+    rev = "v${version}";
+    sha256 = "1g9fyi78nmmm19l2cggwj14m4n80rz7gmnh1gq376zids71s6qxv";
+  };
+
+  useDune2 = true;
+
+  propagatedBuildInputs = [
+    gmp
+    cstruct
+    bigstring
+    dune-configurator
```

Comment:
I guess that `dune-configurator` need not be propagated.

Reply:
Thank you, moved it to `buildInputs` since it won't build without it